<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HERMESS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>600</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="logTab">
     <attribute name="title">
      <string>Log</string>
     </attribute>
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="configurationBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Configuration</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QCheckBox" name="outputCheck">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="plotCheck">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
      <widget class="QComboBox" name="canInterfaceCombo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>ixxat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kvaser</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="canChannelCombo">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="canBitrateCombo">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>500000</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="saveCheck">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>20</y>
         <width>70</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto-Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="canInterfaceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Interface</string>
       </property>
      </widget>
      <widget class="QLabel" name="canChannelLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Channel</string>
       </property>
      </widget>
      <widget class="QLabel" name="canBitrateLabel">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>50</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Bitrate (bps)</string>
       </property>
      </widget>
      <widget class="QLabel" name="supVoltageLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Supply Voltage</string>
       </property>
      </widget>
      <widget class="QLabel" name="kFactorLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>k-Factor</string>
       </property>
      </widget>
      <widget class="QLabel" name="latStrainLabel">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Lateral Strain</string>
       </property>
      </widget>
      <widget class="QLabel" name="meRangeLabel">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>110</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Measuring Range</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="supVoltageEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>1.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="kFactorEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>130</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>4.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="latStrainEdit">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="meRangeEdit">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>130</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0.065536</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QCheckBox" name="absStrainCheck">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>101</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Absolute Strain</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="actionBox">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>381</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Actions</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QLabel" name="tempSRLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sampling Rate (Hz) (Temperature)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="tempSRCombo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string extracomment="Hz">10</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="dmsSRLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Sampling Rate (Hz) (DMS)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="dmsSRCombo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="tareButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Tare</string>
       </property>
      </widget>
      <widget class="QPushButton" name="setSR_DMSButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Set SR DMS</string>
       </property>
      </widget>
      <widget class="QPushButton" name="setSR_TEMPButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>90</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Set SR TEMP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="setFF_DMSButton">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>40</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Set FF DMS</string>
       </property>
      </widget>
      <widget class="QLabel" name="dmsFFLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Filter Frequency (Hz) (DMS)</string>
       </property>
      </widget>
      <widget class="QComboBox" name="dmsFFCombo">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>40</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>36</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>144</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>196</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>256</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>324</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>400</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="tareLabel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>70</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset Sensors</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="experimentBox">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>140</y>
        <width>381</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>Experiment</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QTextEdit" name="documentationText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>121</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Results</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>70</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type of Force</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>90</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Type of Test</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Applied Force</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>40</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Level Arm</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>231</width>
         <height>201</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>ressources/experiment.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>510</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testPlotButton">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>480</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Test Plot</string>
      </property>
     </widget>
     <widget class="Line" name="buttonLine">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>460</y>
        <width>811</width>
        <height>31</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="outputBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>381</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Ouput</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>171</width>
         <height>261</height>
        </rect>
       </property>
       <property name="title">
        <string>DMS</string>
       </property>
       <widget class="QLabel" name="dms_7Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="dms_8Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="dms_6Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="dms_5Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="dms_4Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="dms_2Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="dms_1Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="dms_3Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>171</width>
         <height>261</height>
        </rect>
       </property>
       <property name="title">
        <string>Temperature</string>
       </property>
       <widget class="QLabel" name="temp_6Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="temp_7Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="temp_8Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="temp_2Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="temp_4Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="temp_1Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="temp_5Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="temp_3Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="kalTab">
     <attribute name="title">
      <string>Calibration</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
